docker/docker.go

```diff
@@ -145,6 +155,24 @@ func PullImage(dClient *client.Client, hostname string, containerImage string) e
 	return nil
 }
 
+func UseLocalOrPull(dClient *client.Client, hostname string, containerImage string, plane string) error {
+	logrus.Infof("[%s] Checking image [%s] on host [%s]", plane, containerImage, hostname)
+	imageExists, err := localImageExists(dClient, hostname, containerImage)
+	if err != nil {
+		return err
+	}
+	if !imageExists {
```

Comment:
Nit: better to use if imageExists { return nil} construct. Always better to check the positive